### ISSUE TITLE
Link to New User Profile page broken

### ISSUE THREAD

**Issue by bronwen9** [publiclab/plots2#2447]
New user page: 
https://publiclab.org/notes/Charlescollins83/03-08-2018/Charlescollins83 is unavailable. 

They've made one post at: https://publiclab.org/notes/Charlescollins83/03-08-2018/tvi, which is in the moderation queue.

**Comment by jywarren** [comment]
Hi, it looks like the profile link is wrong -- where did the link come from? Thanks for reporting! I think if we fix the link to https://publiclab.org/profile/Charlescollins83 we should be good 👍

**Comment by bronwen9** [comment]
This was a spam author who I wanted to ban: it was linked from the
moderator email.

**Comment by jywarren** [comment]
I'm having some trouble finding the wrong link to identify the bad code -- if you could try to see where in the email the link was from we can fix it -- thanks, and no rush!

**Comment by jywarren** [comment]
Here's one of the templates! https://github.com/publiclab/plots2/blob/master/app/views/admin_mailer/notify_moderators_of_spam.html.erb

**Comment by bronwen9** [comment]
The email came through the moderators list on March 7, SUbject was: [New Public Lab poster needs moderation] TVI

Content (copy/pasted from email)
First-time poster Charlescollins83 has submitted their first research note!
Please help to approve it or mark it as spam: https://publiclab.org/notes/Charlescollins83/03-08-2018/tvi
Approve or Spam
test
It has the following tags:
User Info
--
527011
Charlescollins83
charlescollins83@hotmail.com

I'll forward it to you as well. I feel like I've seen this occasionally before. Thanks!

**Comment by bronwen9** [comment]
Ah! I *just* clicked on the profile again, and see that it is showing up now (maybe you fixed it?) I am going to go ahead and ban it, thanks!

**Comment by bronwen9** [comment]
Ohh, wait, scratch that, now I remember that I don't think it's spam after all, but kind of wanted to check the profile to double check. The navigation just got weirder. When I clicked in from the email it worked, but when I tried to click to the profile from the post it shows up as the error.
![collins832](https://user-images.githubusercontent.com/8331717/37233113-e70c16b0-23bf-11e8-9308-985c4c4eaa39.gif)
